docs/03-pages/01-building-your-application/08-deploying/04-ci-build-caching.mdx

```diff
@@ -87,6 +87,20 @@ with:
     ${{ runner.os }}-nextjs-${{ hashFiles('**/package-lock.json') }}-
 ```
 
+If you'd prefer to use bun instead of npm or node:
+
+```yaml
+uses: actions/cache@v3
+with:
+  path: |
+    ${{ github.workspace }}/.next/cache
+  # Generate a new cache whenever packages or source files change.
+  key: ${{ runner.os }}-nextjs-${{ hashFiles('**/bun.lockb.json') }}-${{ hashFiles('**/*.js', '**/*.jsx', '**/*.ts', '**/*.tsx') }}
```

Comment:
**question**: does bun generates *.lockb.json ? AFAIK as I see bun only generates bun.lockb

Reply:
Correct, it's even `bun.lockb` in the @JCharante 's PoC repo. I'm not sure why it's `bun.lockb.json` in the PR:
https://github.com/JCharante/nextjs-github-actions-bun/blob/060cc02b8140103833081206a8e15e362b117393/.github/workflows/nextjs.yml#L22